project_event/models/project_task.py

```diff
@@ -251,6 +257,11 @@ def _onchange_room_id(self):
     def _onchange_equipment_id(self):
         self.verify_equipment_bookable()
 
+    @api.onchange('partner_id')
```

Comment:
@jananjoy we already have an onchange method in the parent class, better to override it

Reply:
@rimbendhaou: Overwrite it   based on your comment

Reply:
@rimbendhaou Done